test/validation/api/crypto/test_vectors_len.h

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
> ci: skip_checkformat skip_all
> Change-Id: I360b7f7ec9af1833ca13b4662286bd9a88f61d54

Please remove these from all commit messages.

Reply:
Ack

Reply:
This has not been fixed in v2.

Reply:
I am dropping this patch as it was only required for internal patch which is yet to be in upstream. Thanks for the verification.